app/Kconfig

```diff
@@ -182,6 +182,10 @@ config SM_MQTTC
 config SM_FULL_FOTA
 	bool "Full modem FOTA support"
 
+config SM_FULL_MODEM_DFU
+	bool "Full modem DFU support"
+	default n
```

Comment:
Default is disabled so this line should be removed.
Nordic's build specialists don't allow this line although I personally kind of like it as a documentation.